app/gui/src/dashboard/utilities/vue.ts

```diff
@@ -0,0 +1,23 @@
+/** @file Helpers for using Vue components from React. */
+import * as React from 'react'
+
+import { applyPureVueInReact } from 'veaury'
+import type { AllowedComponentProps, VNodeProps } from 'vue'
+import type { ComponentProps } from 'vue-component-type-helpers'
+
+/** Extracts the properties defined by a component, excluding various Vue internals. */
+type VueComponentProps<T> = Omit<ComponentProps<T>, keyof AllowedComponentProps | keyof VNodeProps>
+
+/** Creates a React component wrapping a Vue component. */
```

Comment:
Jusdging by Docs, it's exactly the same as `applyPureVueInReact`.
```suggestion
/** Creates a Lazy React component wrapping a Vue component. */
```